charts/cloudzero-agent/templates/_helpers.tpl

```diff
@@ -114,8 +114,19 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{/*
 Combine metric lists
 */}}
-{{- define "chartname.combineMetrics" -}}
+{{- define "cloudzero-agent.combineMetrics" -}}
 {{- $total := concat .Values.kubeMetrics .Values.containerMetrics -}}
 {{- $result := join "|" $total -}}
 {{- $result -}}
 {{- end -}}
+
+{{/*
+Required metric labels
+*/}}
+{{- define "cloudzero-agent.requiredMetricLabels" -}}
+{{- $requiredTargetMetricLabels := tuple "__address__" "__scheme__" "__metrics_path__" "__param.*" "__meta.*" "_.*" }}
+{{- $requiredCZMetricLabels := tuple "board_asset_tag" "container" "created_by_kind" "created_by_name" "image" "instance" "namespace" "node" "node_kubernetes_io_instance_type" "pod" "product_name" "provider_id" "resource" "unit" "uid" -}}
```

Comment:
this is kind of an ugly way to store these labels, but I thought it might be a nice thing to "hide" these from the interface that the user sees.